packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/servers/ftp-backup/access/dedicated-server-ftp-backup-access-delete.routing.js

```diff
@@ -0,0 +1,20 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.dedicated-server.server.ftpBackup.deleteAccess', {
+    url: '/deleteAccess',
+    views: {
+      modal: {
+        component: 'dedicatedServerFtpBackupAccessDelete',
+      },
+    },
+    params: {
+      ipbackup: null,
+    },
+    layout: 'modal',
+    resolve: {
+      goBack: /* @ngInject */ (goToFtpBackUpStorage) => goToFtpBackUpStorage,
```

Comment:
Shouldn't reload and banner management be added here in the event of success or error?